### DISCUSSION TITLE
the detector is returning 2 bboxes for image with 2 faces but the smoothing effect is being displayed only on one.

### DISCUSSION THREAD

**Discussion by ZainabZaman** [community/community#57812] [Programming Help]
### Multiple Face Smoothing

Question

@5starkarma I am trying to get smoothing effect for multiple faces using your code provided at https://github.com/5starkarma/face-smoothing but I am getting smoothing for 1 image only if I provide an object with 2 faces. The model detects 2 faces but smoothes only one. 
The loop in the smooth.py returns the bboxes for both the detected faces but smooths only one. Can you guide?

**Comment by 5starkarma** [comment]
It is probably only set up for one. I actually don't know because it has been a couple years since I have touched this. Feel free to use the code in your own project and adjust it how you like. hint: probably need to run both faces through whatever smoothing operations exist
